new-log-viewer/src/utils/config.ts

```diff
@@ -31,6 +31,7 @@ const CONFIG_DEFAULT: ConfigMap = Object.freeze({
  * @param props.key
  * @param props.value
  * @return `null` if the value is valid, or an error message otherwise.
+ * @throws {Error} If the config item should not be handled by the config utilities.
```

Comment:
This is a bit of a nit, but:

```suggestion
 * @throws {Error} If the config item cannot be managed by these config utilities.
```

Ditto below.